synapse/state/__init__.py

```diff
@@ -444,6 +444,21 @@ class _StateResMetrics:
     "expensive room for state resolution",
 )
 
+_cpu_times = Histogram(
+    "cpu",
+    "CPU time (utime+stime) spent computing a single state resolution",
+    namespace="synapse",
+    subsystem="state_res",
```

Comment:
I'd rather we used `synapse_state_res_cpu` name rather than the namespacing stuff, as it makes it *much* easier to grep for (and its consistent with everything else)

Reply:
How about 589eb1e02?